docs/components/button.md

```diff
@@ -275,186 +289,234 @@ The component has several different styles based on its type and color combinati
 :root {
   --button-fill-neutral-border-color: transparent;
   --button-fill-neutral-text-color: var(--c-text-inverse-1);
+  --button-fill-neutral-icon-color: var(--c-text-1);
```

Comment:
Thanks for adding it, but, let's use `text` colors for icons too, since that's what usually Sefirot does. I should have named this `fg` (foreground) instead 😓 That's what it was meant. The color that used on foreground, (which is usually text)

Reply:
Yeah, I tried doing that first 😅. But the issue is the text color for `fill` mode behave differently than `outline`/`text` mode.

Reply:
Ahhh... yeah... true. Thanks! I guess there's no way around this then 😅